tests/foreman/cli/test_errata.py

```diff
@@ -1390,9 +1448,116 @@ def test_positive_user_permission(self):
         @Assert: Check that the new user is able to see errata for one
         product only.
 
-        @caseautomation: notautomated
-
         """
+        user_password = gen_alphanumeric()
+        user_name = gen_alphanumeric()
+        org = self.org3
```

Comment:
this name came from setUpClass when we had to create 3 org, I really have no idea how to name them, (other then org1 , org2, org3) , if you have a good naming proposition it will be very welcome, but will be changed in a separate refactoring commit, as other tests already implemented will be affected.